iOSNativeSwiftTemplate/iOSNativeSwiftTemplate/AppDelegate.swift

```diff
@@ -116,4 +109,96 @@ class AppDelegate : UIResponder, UIApplicationDelegate {
     func enableIDPLoginFlowForURL(_ url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
         return  UserAccountManager.shared.handleIdentityProviderResponse(from: url, with: options)
     }
+    
+    // MARK: - QR Code Login Via Salesforce Identity API UI Bridge Public Implementation
+    
+    /**
+     * When enabling log in via Salesforce UI Bridge API generated QR codes, choose to use the
+     * Salesforce Mobile SDK reference format for QR code log in URLs or an entirely custom format.
+     *
+     * If only one of the two formats is used, which is the most likely implementation for apps using this
+     * template, this variable and code related to the unused implementation could be removed.
+     */
+    let isQrCodeLoginUsingReferenceUrlFormat = true
```

Comment:
@bbirman - I moved this from the SceneDelegate so that the documentation doesn't have the template user updating more files than needed.